### ISSUE TITLE
add option to specify commit message when version number is bumped

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/perennial#86]
I don't know if this belongs in perennial or chipper, so creating an issue here.

Now that bumping the version number is automated, I miss being able to (optionally) add a commit message that describes the purpose of the version.   In the past, I did this for "significant" dev versions, with messages like "before making major changes", "after making major changes", "to verify #13", etc.  This saved me time, making it easy to identify a significant dev version. Now I find that I'm having to keep track of significant dev versions in my notebook, which feels like a step backwards.  My most recent need for this is  "before" and "after' versions for the extensive changes that I'm currently making to Equality Explorer.

I've also done this for a couple of maintenance releases (production versions), where I've indicated the issues that were addressed in the commit message for package.json.  E.g. "fixes #23, #25, #26". This is a much rarer use case (for me), and I could live without it.

So the minimum that would meet my needs would be a new option to `grunt dev`, something like:

`grunt dev --brands=phet --message="before making major changes"`

But perhaps this option would be useful for all tasks that bump version number? I guess that would require some discussion.

The argument to `--message` would be appended to the automated commit message for package.json.  E.g. without `--message`, the package.json commit message is "Bumping version to 1.0.0-dev.63".  With `--message="before making major changes"`, the commit message would be something like "Bumping version to 1.0.0-dev.63, before making major changes".

Assigning to @ariel-phet to prioritize and assign.

**Comment by ariel-phet** [comment]
@jonathanolson said this should be relatively quick. He may have some questions for @pixelzoom

**Comment by jonathanolson** [comment]
> @jonathanolson said this should be relatively quick. He may have some questions for @pixelzoom

More careful reading answered my questions. Running perennial unit tests with changes before I commit/push.

**Comment by jonathanolson** [comment]
Implemented and tests passing. Spot checked 4 different deployment commands. @pixelzoom, can you try a deploy with this?

**Comment by pixelzoom** [comment]
@jonathanolson Can you summarize what you implemented?

**Comment by pixelzoom** [comment]
I had occasion to use this today for `grunt dev`.   Answered the "what did you implement?" question myself by inspecting commit and by looking at `grunt --help` in perennial.  Closing.
